modules/dna_rna_tool.py

```diff
@@ -0,0 +1,58 @@
+TRANSCRIBE_DICT = dict(A='A', T='U', G='G', C='C',
+                       a='a', t='u', g='g', c='c')
+DNA_COMPLEMENT_DICT = dict(A='T', T='A', G='C', C='G',
+                           a='t', t='a', g='c', c='g')
+RNA_COMPLEMENT_DICT = dict(A='U', U='A', G='C', C='G',
+                           a='u', u='a', g='c', c='g')
+
+def check_input_for_nucleic_acids(nucleic_acid: str) -> bool:
+    """
+    Function checks if inputed data is a nucleic acid or not. If it is it
+    returns True, else False.
+    :param nucleic_acid: inputed data, type:str
+    :return: True if inputed data is a nucleic acid, else returns False
+    """
+    dna_rna_alphabet = set('ATUGCatugc')
+    if (dna_rna_alphabet.issuperset(nucleic_acid)):
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    return dna_rna_alphabet.issuperset(nucleic_acid):
```
Не надо проверять результат булевой операции чтобы вернуть булево значение, можно сразу возвращать булевый результат